src/main/kotlin/frc/robot/subsystems/intake/IntakeIOReal.kt

```diff
@@ -60,22 +85,14 @@ class IntakeIOReal : IntakeIO {
         angleMotor.setPosition(0.0)
     }
 
+    override fun setGains(kP: Double, kI: Double, kD: Double) {
+
+    }
+
     override fun updateInputs() {
         inputs.angleMotorAngle = Units.Rotations.of(angleMotor.position.value)
```

Comment:
Why not mutable?

Reply:
Why would an input be mutable? Also I don't think it's relevant to this PR.

Reply:
Inputs change all the time, currently you create new objects every loop
But this can be fixed later